readme.md

```diff
@@ -181,9 +181,19 @@ Next, you'll need to declare four environment variables in your CI. Open the
 settings page for your CI provider.
 
 - **GitHub Actions**:
-  https://github.com/olafurpg/sbt-ci-release/settings/secrets
-- **Travis CI**: https://travis-ci.com/scalameta/sbt-scalafmt/settings (Make
-  sure that "Build pushed branches" setting is enabled).
+
+  Select `Settings -> Secrets -> New repository secret` to add each of the
+  required variables as shown in the next figure:
+
+  ![secrets button 2021-01-27](docs/github-secrets-2021-01-27.png)
```

Comment:
These images look empty. Can you upload them to a github issue and use the link instead? My experience is that it's best to avoid committing image files into the git repository.

Reply:
Ok, I understand about not checking images in. I didn't know you could upload them to an issue so I didn't want them on my personal github - I will look into that. It'll take me a couple of days to get to this. Thanks for reviewing.

Reply:
This seems ok. I can't seem to understand how to upload a file that is associated with this repo.
https://github.com/olafurpg/sbt-ci-release/tree/c0febea24fe111f80933824656eff90cea42baa9